File: Backend/ExamSupportToolAPI/ExamSupportToolAPI.DataObjects/StudentPresentationForStudent.cs

namespace ExamSupportToolAPI.DataObjects
{
    public class StudentPresentationForStudent
    {
        public int Status { get; set; }
        public decimal TheoryGrade { get; set; }
        public decimal ProjectGrade { get; set; }
        public DateTime? StartingTime { get; set; }
        public ExaminationTicketForStudent? ExaminationTicket { get; set; }
    }
}

File: Frontend/ExamSupportToolUI/src/app/shared/models/student/student-presentation-for-student.ts
import { ExaminationTicketForStudent } from "./examination-ticket-for-student";

export interface StudentPresentationForStudent {
    status: number;
    startingTime?: Date;
    theoryGrade: number;
    projectGrade: number;
    examinationTicket: ExaminationTicketForStudent;
}